### PR TITLE
feat(Chat): add Control messages prototype

### PR DESCRIPTION
**Pull request by mnajdova**
This PR adds prototype for the Control messages that can be used in the Chat. They are based on the Tree* components, with some custom overrides on the behaviors. The styling is not done currently, we want to focus on the accessibility first and test all scenarios.

### CODE REVIEW

File: docs/src/prototypes/chatMessages/ControlMessages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        },
+        content: (
+          <Tree
+            data-is-focusable="true"
+            accessibility={overridenTreeBehavior}
+            items={[
+              {
+                onKeyDown: e => {
+                  // Without this override, the focus zone for the tree title is not working..
+                  console.log('Tree item 1 key down')
```
Comment (mnajdova) [inline docs/src/prototypes/chatMessages/ControlMessages/index.tsx]: The inner Focus zone doesn't work if I remove this event handler.. I tried just adding data-is-focusable, tabIndex: 0, but these two didn't help..

File: docs/src/prototypes/chatMessages/ControlMessages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+          const eventCode = keyboardKey.getCode(e)
+          if (eventCode === keyboardKey.Enter || eventCode === keyboardKey.Spacebar) {
+            setOpen(true)
+          }
+          if (eventCode === keyboardKey.Escape) {
+            setOpen(false)
+          }
+        },
+        content: (
+          <Tree
```
Comment (lucivpav) [inline docs/src/prototypes/chatMessages/ControlMessages/index.tsx:113]: Looking at the chat example, it was not clear to me that this is actually an expansible tree.

File: docs/src/prototypes/chatMessages/ControlMessages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: {
+    color: 'green',
+    icon: 'check',
+  },
+}
+
+const treeItems = [
```
Comment (lucivpav) [inline docs/src/prototypes/chatMessages/ControlMessages/index.tsx:24]: These two items are almost identical. How about extracting the code into a function?

File: docs/src/prototypes/chatMessages/ControlMessages/index.tsx (hunk 2 above)
Comment (mnajdova) [reply docs/src/prototypes/chatMessages/ControlMessages/index.tsx:113]: The styling are not implemented for this prototype, we are focusing only on the behavior and whether everything works as expected

File: docs/src/prototypes/chatMessages/ControlMessages/index.tsx (hunk 3 above)
Comment (mnajdova) [reply docs/src/prototypes/chatMessages/ControlMessages/index.tsx:24]: will refactor it once we decide on the final implementation

**Comment by jdhuntington** [comment]
Closing as this has seen no activity for awhile. Please reopen if still relevant. Thanks!

**Pull request closed without merging**